_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-tactile
```

Comment:
I've never really messed around with the Jekyll themes, I probably would keep these off when working on a daily project but if you are experiments that is cool, I would just experiment on a branch that isn't master that way you always have a working version that is "production" or live to the public.

Reply:
Got it. I actually don't quite know what this does! Was just playing around with settings

Reply:
oh yeh I figured all it does it kinda like sets up the theme and builds a mini webpage like maybe Wordpress or Wix would do in a sense... Jekyll is essentially a JS / Gatsby based library that is involved with static templates, and blog style pages, and similar items.